src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReplacementServiceTests.cs

```diff
@@ -873,6 +874,861 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbC
+                var replacementPlan = result.Right;
+                Assert.False(replacementPlan.Valid);
+
+                Assert.Single(replacementPlan.DataBlocks);
+                var dataBlockPlan = replacementPlan.DataBlocks.First();
+                Assert.False(dataBlockPlan.Valid);
+                Assert.False(dataBlockPlan.Fixable);
+            }
+        }
+
+        [Fact]
+        public async Task GetReplacementPlan_NewFiltersIntroduced_ReplacementInvalidAndNotFixable()
+        {
+            var originalSubject = new Subject
+            {
+                Id = Guid.NewGuid()
+            };
+
+            var replacementSubject = new Subject
+            {
+                Id = Guid.NewGuid()
+            };
+
+            var contentRelease = new Content.Model.Release
+            {
+                Id = Guid.NewGuid(),
+                PreviousVersionId = null
+            };
+
+            var statsRelease = new Release
+            {
+                Id = contentRelease.Id,
+                PreviousVersionId = contentRelease.PreviousVersionId
+            };
+
+            var originalFile = new File
+            {
+                Filename = "original.csv",
+                Type = FileType.Data,
+                SubjectId = originalSubject.Id
+            };
+
+            var replacementFile = new File
+            {
+                Filename = "replacement.csv",
+                Type = FileType.Data,
+                SubjectId = replacementSubject.Id
+            };
+
+            var originalReleaseFile = new ReleaseFile
+            {
+                Release = contentRelease,
+                File = originalFile
+            };
+
+            var replacementReleaseFile = new ReleaseFile
+            {
+                Release = contentRelease,
+                File = replacementFile
+            };
+
+            var originalDefaultFilterItem = new FilterItem
+            {
+                Id = Guid.NewGuid(),
+                Label = "Test filter item"
+            };
+
+            var replacementDefaultFilterItem = new FilterItem
+            {
+                Label = "Test filter item"
+            };
+
+            var replacementNewlyIntroducedFiltersFilterItem = new FilterItem
+            {
+                Label = "Filter item for newly introduced Filter"
+            };
+
+            var originalDefaultFilterGroup = new FilterGroup
+            {
+                Label = "Default group - not changing",
+                FilterItems = new List<FilterItem>
+                {
+                    originalDefaultFilterItem
+                }
+            };
+
+            var replacementDefaultFilterGroup = new FilterGroup
+            {
+                Label = "Default group - not changing",
+                FilterItems = new List<FilterItem>
+                {
+                    replacementDefaultFilterItem,
+                    replacementNewlyIntroducedFiltersFilterItem
```

Comment:
Should this be here @duncan-at-hiveit as I think you might have filter items belonging to two different filter groups. See also line 1524.